bitnami/minio/2024/debian-11/rootfs/opt/bitnami/scripts/minio-env.sh

```diff
@@ -78,6 +78,7 @@ export MINIO_DISTRIBUTED_MODE_ENABLED="${MINIO_DISTRIBUTED_MODE_ENABLED:-no}"
 export MINIO_DEFAULT_BUCKETS="${MINIO_DEFAULT_BUCKETS:-}"
 export MINIO_STARTUP_TIMEOUT="${MINIO_STARTUP_TIMEOUT:-10}"
 export MINIO_SERVER_URL="${MINIO_SERVER_URL:-$MINIO_SCHEME://localhost:$MINIO_API_PORT_NUMBER}"
+export MINIO_DATA_DIR="${MINIO_DATA_DIR:-/bitnami/minio/data}"
```

Comment:
Minor nitpick; could you move it back to the path section? It's true it could also be considered part of MinIO config but having all the path-related envs grouped seems more intuitive to me.

Reply:
Sure! I was 50/50 on where this should go in here as everything in the "path" section is non-configurable, but this is still a path, but now it is a configurable path!

Reply:
Done

Reply:
@FraPazGal Bumping this

Reply:
Perfect, the PR LGTM @Mo0rBy! Let me run a couple of check in our internal pipeline before merging this.